src/nodes/Geometry3D/IndexedFaceSet.js

```diff
@@ -791,8 +791,21 @@ x3dom.registerNodeType(
                         hasColor = false;
                     }
                     this._mesh._numColComponents = numColComponents;
+                    if (fieldName.startsWith("attrib")) {
+                        var attrName = fieldName.slice(7);
```

Comment:
if you call node.updateAttrib in the IFS, the only information given is FieldChanged("attribs").
A solution would be to write the function node.updateAttrib(name) in FloatVertexAttribute.js, and call it before (or after) the call of FieldChanged on IndexedFaceSet.